android/julius/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.github.bvschaik.julius">
+
+    <uses-feature android:glEsVersion="0x00020000"/>
+    <uses-feature android:name="android.hardware.touchscreen" android:required="false" />
+    <uses-feature android:name="android.hardware.gamepad" android:required="false" />
+    <uses-feature android:name="android.hardware.audio.output" android:required="false" />
+    <uses-feature android:name="android.hardware.type.pc" android:required="false" />
+
+    <application
+        android:allowBackup="true"
+        android:label="Julius"
+        android:icon="@mipmap/julius"
+        android:theme="@android:style/Theme.NoTitleBar.Fullscreen"
+		android:hardwareAccelerated="true">
+
+        <activity
+            android:name=".JuliusMainActivity"
+            android:label="Julius"
+            android:alwaysRetainTaskState="true"
+            android:launchMode="singleInstance"
+            android:resizeableActivity="true"
+            android:configChanges="orientation|uiMode|density|screenLayout|screenSize|smallestScreenSize|keyboard|keyboardHidden|navigation">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+
+        <meta-data android:name="com.samsung.android.keepalive.density" android:value="true" />
```

Comment:
If I remember well, it's needed to get the screen dpi

Reply:
It shouldn't. This property is Samsung-specific. I'd rather not have vendor specific code in the codebase.

Reply:
Oh you're right, I remember now, I added this to be able to play Julius using Samsung DeX!

Julius is fully compatible with DeX and runs very well, so can we please keep it?

Reply:
DeX support would be great.

Reply:
At least add a comment explaining why we need this. What the heck is DeX?

Reply:
I'll have at look if it still works fine without the line (it might just force close the app when switching in and out of DeX without that line), if it still works then the line can be safely removed.

Reply:
DeX is a PC-like mode for android when connected to a monitor using HDMI or to a PC using the USB interface and the Samsung DeX app for Windows).

With DeX, you can play Caesar 3 on Android as you would on a regular PC:

![dex](https://user-images.githubusercontent.com/6518876/81160996-c4e3b800-8f82-11ea-80e3-b0160b13940e.JPG)